core/vm/interpreter.go

```diff
@@ -137,12 +137,17 @@ func (in *Interpreter) Run(snapshot int, contract *Contract, input []byte) (ret
 		// to be uint256. Practically much less so feasible.
 		pc   = uint64(0) // program counter
 		cost uint64
+		// copies used by tracer
+		stackCopy = newstack() // stackCopy needed for Tracer since stack is mutated by 63/64 gas rule 
+		pcCopy = uint64(0) // needed for the deferred Tracer
+		gasCopy = uint64(0) // for Tracer to log gas remaining before execution
+		logged = bool(false) // deferred Tracer should ignore already logged steps
```

Comment:
Please use 

```go
 		// copies used by tracer
         stackCopy = newstack() // needed for Tracer since stack is mutated by 63/64 gas rule 
 		pcCopy    uint64       // needed for the deferred Tracer
 		gasCopy   uint64       // for Tracer to log gas remaining before execution
 		logged    bool         // deferred Tracer should ignore already logged steps
```